File: scripts/check-dist.py
from pathlib import Path

ROOT = Path(__file__).resolve().parent.parent
DIST = ROOT / "dist"
REQUIRED = ["index.html", "styles.css", "bg_logo.jpg"]

missing = [str(file) for file in (DIST / filename for filename in REQUIRED) if not file.exists()]

if missing:
    print("Build verification failed. Missing files:")
    for file in missing:
        print(f"- {file}")
    raise SystemExit(1)

print("✅ Build verification passed. All expected files exist in dist/")

File: scripts/check-dist.js
const fs = require('fs');
const path = require('path');

const files = ['dist/index.html', 'dist/styles.css', 'dist/bg_logo.jpg'];
const missing = files.filter((file) => !fs.existsSync(path.resolve(file)));

if (missing.length) {
  console.error('Build verification failed. Missing files:');
  missing.forEach((file) => console.error(`- ${file}`));
  process.exit(1);
}

console.log('✅ Build verification passed. All expected files exist in dist/');
